instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py

```diff
@@ -119,38 +119,11 @@ def __init__(
             self._config = _DEFAULT_CONFIG
```

Comment:
the original attribute `labels` defined in line 114 is still needed?

also, these deleted labels are used later on in this file, inside the functions, should they be deleted as well?